util/integrations/coordinape/etl.js

```diff
@@ -0,0 +1,45 @@
+import { pinJSONToIPFS } from '../../pinata'
+import { findCommunityByAddress, updateCommunityDataHash } from '../../fauna'
+import { getIpfsData } from '../../ipfs'
+import { transform } from './transform'
+import { load } from './load'
+
+const etl = async (address, epochData) => {
+  // Get Community:
+  const { _id: id, membersHash } = await findCommunityByAddress(address)
+
+  // Load prior IPFS data
+  let membersData = {}
+  if (membersHash) {
+    membersData = await getIpfsData(membersHash)
+  }
+
+  // Extract data:
+  // const { name } = epochData
```

Comment:
Same to remove, or leaving it for documentation reasons?

Reply:
I have a little cleanup here and elsewhere yet I think.